src/components/App.tsx

```diff
@@ -44,11 +57,10 @@ const PrivateRoute: React.FC<RouteProps> = ({ component, ...rest }) => {
           render={(props): JSX.Element => {
             if (!token) {
               if (
-                process.env.REACT_APP_AUTH_TYPE === 'external' &&
-                process.env.REACT_APP_EXTERNAL_AUTH_LOGIN_URL
+                EXTERNAL_AUTH === true &&
```

Comment:
Could these be changed to just
```suggestion
                EXTERNAL_AUTH &&
```

Reply:
I changed this as per your suggestion in the commit below.